src/views/Settings.vue

```diff
@@ -263,7 +263,11 @@ export default defineComponent({
       appVersion: "",
       locales: process.env.VUE_APP_LOCALES ? JSON.parse(process.env.VUE_APP_LOCALES) : {"en": "English"},
       currentFacilityDetails: {} as any,
-      isStoreFulfilmentTurnedOn: true
+      facilityGroupDetails: {} as any,
+      outstandingOrdersCount: 0,
```

Comment:
Why we have defined the variables for order count?

Reply:
Removed sir